test/e2e/framework/util.go

```diff
@@ -2668,13 +2693,7 @@ func WaitForAllNodesSchedulable(c clientset.Interface, timeout time.Duration) er
 		}
 		for i := range nodes.Items {
 			node := &nodes.Items[i]
-			if _, hasMasterRoleLabel := node.ObjectMeta.Labels[service.LabelNodeRoleMaster]; hasMasterRoleLabel {
-				// Kops clusters have masters with spec.unscheduable = false and
-				// node-role.kubernetes.io/master NoSchedule taint.
-				// Don't wait for them.
-				continue
-			}
-			if !isNodeSchedulable(node) || !isNodeUntainted(node) {
+			if !isNodeSchedulable(node) || isNodeConditionallyTainted(node) {
```

Comment:
isNodeConditionallyTainted is a confusing name. suggest something like `nodeHasKnownNoScheduleTaints`

Reply:
Sure, rename is fine to me.